### DISCUSSION TITLE
Disabling UI Command Menu

### DISCUSSION THREAD

**Discussion by iskaa02** [vscode-neovim/vscode-neovim#1960] [Q&A] [answered]
Is it possible to disable the UI Command menu and use the default behavior for commands (cmdline,search) which is in the statusbar?

the docs say nothing about this, and no discussions or issues seems to be related.

**Comment by justinmk** [comment]
No. Do you want the command to show in the vscode statusbar, or are you expecting vscode-neovim to draw a quasi-statusbar at the bottom of the buffer (vscode document) text?

**Comment by iskaa02** [comment]
i want the commands to be in the vscode statusbar, or at least move the menu to the bottom.
I regularly use neovim and vscode-neovim, and i want to keep it as similar as possible when changing context, including mappings and command placement.

**Reply by theol0403** [reply]
The choice was made between supporting completions and being in the bottom bar.

**Comment by justinmk** [comment, marked as answer]
> or at least move the menu to the bottom.

That sounds like a feature request for vscode: https://github.com/microsoft/vscode/issues/17268

 Not something that vscode-neovim can or should solve.
